4001-5000/LIT4633Chronicle.xml

```diff
@@ -1020,76 +1042,91 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 
                 <div type="textpart" subtype="paragraph" n="59">
                     <ab>1ክፍል፡ ፳፪።2ሠረቀ፡ ግንቦት፡ በዕለተ፡ እሑድ።3ወበውእቱ፡ ወርኅ፡ ቦአ፡ ኑፋቄ፡ (A160vb) ውስተ፡
-                        ልበ፡ ሰብእ፡ በእንተ፡ ዐመፃሁ፡ ለአምኃ፡ ኢየሱስ።
-                        
+                        ልበ፡ ሰብእ፡ በእንተ፡ ዐመፃሁ፡ ለ<persName ref="PRS12774AmmehaIyasus">አምኃ፡ ኢየሱስ።
+                        </persName>                        
                     </ab>
                 </div>
                 
                 <div type="textpart" subtype="paragraph" n="60">
-                    <ab>1ክፍል፡ ፳፫።2ሠረቀ፡ ሰኔ፡ በዕለተ፡ ሠሉስ።3ወበወእቱ፡ ወርኅ፡ ተገብረ፡ ሎቱ፡ ለዐቢይ1፡ ንጉሥ፡ 
-                        በካፋ፡ ዐቢይ2፡ ኀይል3፡ እምዐቢይ4፡ እግዚአብሔር፡ ዘዐቢይ5፡ ኀይሉ6።4ምክንያቱሰ፡ አንሥአ፡
-                        ከተማ፡ ነገረ፡ ዘውእቱ፡ ወሬ፡ እንዘ፡ ይብል፡ የሐውር፡ ንጉሥ፡ ውስተ፡ ወልቃይት።5ወበዝ፡ ተስፋ፡ 
-                        ቦአ፡ ወሬኛ፡ ውስተ፡ መዲና፡  ወተኀብአ7፡ በቤተ፡ አዛዢ፡  ዘርአ፡ ጽዮን።6ወአኀዘ8፡ ይስሕብ፡ ሰብአ፡ እንዘ፡ ይብል፡ 
-                        ናሁ፡ እነግሥ፡ አነ፡ እስመ፡ በጽሐ፡ ጊዜ፡ መንግሥትየ።7ወእሁበክሙ፡ ሢመተ፡ 
-                        ወሢመተ፡ እገሌ፡ እሁብ፡ ለእገሌ።8ወበዝ፡ ኀብሩ9፡ ሎቱ፡ ብዙኃን፡ ሰብእ፡ ፈድፋደሰ፡ ወዓልቱ፡ ለማሙዬ።9
+                    <ab>1ክፍል፡ ፳፫።ሠረቀ፡ ሰኔ፡ በዕለተ፡ ሠሉስ።3ወበወእቱ፡ ወርኅ፡ ተገብረ፡ ሎቱ፡ ለዐቢይ፡
+                        <persName ref="PRS2377Bakaffa"><roleName type="title">ንጉሥ፡
+                        </roleName>በካፋ፡ </persName> 
+                        ዐቢይ፡ ኀይል፡ እምዐቢይ፡ እግዚአብሔር፡ ዘዐቢይ፡ ኀይሉ።ምክንያቱሰ፡ አንሥአ፡
+                        ከተማ፡ ነገረ፡ ዘውእቱ፡ ወሬ፡ እንዘ፡ ይብል፡ የሐውር፡   <persName ref="PRS2377Bakaffa"><roleName type="title">ንጉሥ፡
+                        </roleName>  </persName> ውስተ፡ <placeName ref="LOC6186Walqay">ወልቃይት።</placeName> ወበዝ፡ ተስፋ፡ 
+                        ቦአ፡ ወሬኛ፡ ውስተ፡ መዲና፡  ወተኀብአ፡ በቤተ፡  <persName ref="PRS12807ZaraSeyon"><roleName type="title">አዛዢ፡</roleName> ዘርአ፡ ጽዮን።</persName> 
+                        ወአኀዘ፡ ይስሕብ፡ ሰብአ፡ እንዘ፡ ይብል፡ 
+                        ናሁ፡ እነግሥ፡ አነ፡ እስመ፡ በጽሐ፡ ጊዜ፡ መንግሥትየ።ወእሁበክሙ፡ ሢመተ፡ 
+                        ወሢመተ፡ እገሌ፡ እሁብ፡ ለእገሌ።ወበዝ፡ ኀብሩ፡ ሎቱ፡ ብዙኃን፡ ሰብእ፡ ፈድፋደሰ፡ ወዓልቱ፡ ለ<persName ref="PRS12775Mammuye">ማሙዬ።
+                        </persName>
```

Comment:
There are no spaces after ።, not only here but again throughout the record

Reply:
I have corrected now.